.circleci/config.yml

```diff
@@ -718,6 +718,39 @@ jobs:
           command: yarn test:ci
       - magma_slack_notify
 
+  nms-e2e-test:
+    machine:
+      image: ubuntu-1604:201903-01
+    environment:
+      - NMS_ROOT=${MAGMA_ROOT}/nms/app/packages/magmalte
+    steps:
+      - checkout
+      - docker/install-dc
+      - run: sudo apt-get update -y
+      - run: sudo apt-get install -y apt-transport-https
+      - run: curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+      - run: echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+      - run: sudo apt-get update -y
+      - run: sudo apt-get install -y yarn
+      - run: echo 'export MAGMA_ROOT=$(pwd)' >> $BASH_ENV
+      - run:
+          command: |
+            source $NVM_DIR/nvm.sh
+            cd ${MAGMA_ROOT}/nms/app
+            nvm install stable
+            yarn install
+            cd ${MAGMA_ROOT}/nms/app/packages/magmalte
+            ./e2e_test_setup.sh
+            cd ${MAGMA_ROOT}/nms/app
+            yarn test:e2e
+            cd ${MAGMA_ROOT}/nms/app/packages/magmalte
+            docker-compose -f docker-compose-e2e.yml logs magmalte &> /tmp/nms_artifacts/magmalte.log
+            docker-compose -f docker-compose-e2e.yml logs mock_server &> /tmp/nms_artifacts/mock_server.log
```

Comment:
Can most of these commands be put into the `e2e_test_setup.sh` script? If so I think it would be better to do that than have a bunch of bash commands in the CI config